pkg/kube/configmapwatcher/configmapwatcher.go

```diff
@@ -64,10 +64,7 @@ func NewController(client kube.Client, namespace, name string, callback func(*v1
 		Core().V1().ConfigMaps()
 
 	c.queue = controllers.NewQueue("configmap "+name, controllers.WithReconciler(c.processItem))
-	c.informer.Informer().AddEventHandler(controllers.FilteredObjectSpecHandler(c.queue.AddObject, func(o controllers.Object) bool {
-		// Filter out configmaps
-		return o.GetName() == name && o.GetNamespace() == namespace
```

Comment:
why remove the filter?

Reply:
see above L60-L62, ns/name  is specified there

Reply:
Oh yeah. Wonder why we had that

Reply:
I know why we need it here. The tests failed because k8s fake client list-watch does not support field selector yet